### PR TITLE
Switch Foreman log level to debug for hammer tests

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: fb-hammer-tests.bats [hunk 1]
```diff
@@ -20,6 +20,14 @@ LOG_DIR=/root/hammer_test_logs
   gem install open4 colorize
 }
 
+@test "switch forman log level to debug" {
+  sed -i 's/.*config.log_level.*/  config.log_level = :debug/g' /usr/share/foreman/config/environments/production.rb
```
Comment (ekohl) [inline fb-hammer-tests.bats]: This can't be done though a config file?

Comment (domcleal) [inline fb-hammer-tests.bats]: Nope.. http://projects.theforeman.org/issues/6524

File: fb-hammer-tests.bats [hunk 2]
```diff
@@ -20,6 +20,14 @@ LOG_DIR=/root/hammer_test_logs
   gem install open4 colorize
 }
 
+@test "switch forman log level to debug" {
```
Comment (domcleal) [inline fb-hammer-tests.bats]: "Foreman"

File: fb-hammer-tests.bats (hunk 1 above)
Comment (ekohl) [inline fb-hammer-tests.bats]: I wouldn't mind if this was a comment in the code.

**Comment by mmoll** [comment]
díky @tstrachota!

**Pull request merged**